packages/static-hosting/lib/static-hosting.ts

```diff
@@ -560,6 +560,31 @@ export class StaticHosting extends Construct {
       }
     }
 
+    if (enableStaticFileRemap) {
+      const staticFileRemapPrefixes = props.defaultBehaviourPrefixes?.map(
+        prefix => `${prefix.prefix}/`
+      ) || [""];
```

Comment:
Just to confirm you've tested this without passing in any default behaviour prefixes?